Bottom: 0d3061a4b70216e4fa97849de2d386c12235be4e
Top:    c11c3449ead7488edd95fa44ab5af9964c3fc223
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-16 19:47:20 +0800

Refresh of vhost_net-rx-batching

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index f7ab7e8..c7f3922 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -103,7 +103,7 @@ struct vhost_net_virtqueue {
 	 * Protected by vq mutex. Writers must also take device mutex. */
 	struct vhost_net_ubuf_ref *ubufs;
 	struct skb_array *rx_array;
-	struct sk_buff *skbs[VHOST_RX_BATCH];
+	void *skbs[VHOST_RX_BATCH];
 	int nskbs;
 	int skb_index;
 };
